docs/index.md

```diff
@@ -0,0 +1,98 @@
+## Overview
+This guide walk-through the process of deploying an Sample Java application in CFT.
+
+This guide uses a pre-configured application from a template. Application is deployed to a kubernetes cluster. 
+At the end of this tutorial, you should be able to access an working application via VPN and made changes to it.
+
+## Prerequisite
+
+`Github` access is required to complete the steps in this tutorial. Following the [link](https://hmcts.github.io/onboarding/team/github.html#github) to request access.
+
+## High Level Steps
```

Comment:
Backstage automatically generates a Table of contents so I think this whole section can be removed

![image](https://user-images.githubusercontent.com/21194782/143414115-9db2d0c7-c42d-43c0-b756-114aa72c32e6.png)